Assets/Python/GreatPeople.py

```diff
@@ -1641,16 +1940,20 @@ def setup():
 		"Thomas Hobbes", # 17th
 		"Robert Walpole", # 18th
 		"William Pitt", # 18th
+		"fMary Wollstonecraft", # 18th
 		iIndustrial,
 		"William Gladstone", # 19th
 		"Benjamin Disraeli", # 19th
 		iGlobal,
+		"Thomas Edward Lawrence", # 20th
 		"fEmmeline Pankhurst", # 20th
 		"Clement Atlee", # 20th
+		"fDiana Spencer", # 20th
 	],
 	iGreatGeneral : [
 		"William the Conqueror", # 11th
 		"Richard the Lionheart", # 12th
+		"William Wallace", # 13th
```

Comment:
I don't think Scottish figures before unification should be included (especially those who fought against England)